functions_library.sh

```diff
@@ -41,17 +41,17 @@ feedback() {
 change_branch() {
   # first and only parameter is the branch to checkout
 
-  # -z tests for zero length.
-    if [ -z ${1+x} ]; then
-        echo "Working from main branch.";
-    else
+  # -z tests for zero length. 
```

Comment:
For the love of all holy, please, turn off your white space editing.

Reply:
It's not a big deal, but it makes reviewing edits a lot easier.

Reply:
There are no mods on the `functions_library.sh` file, so I can't see what you're seeing.

Second, I don't have any white space editing option thing going on my editor. I don't do space-
editing, I always prefer tabs.

Reply:
Seems to have disappeared.

Reply:
I did yesterday night a reverse-commit so maybe you were seeing the old mods.